metabase/metabase
# Mantine TextInput and NumberInput

How to verify:
- `yarn storybook`
- Open http://localhost:6006/?path=/docs/inputs-textinput--default-story
- Compare with https://www.figma.com/file/oIZhYS5OoRA7twd4KqN4Eu/Input-%2F-Text?type=design&node-id=5-70&mode=design&t=dsUIfXbrSyAZT8ku-0

Note that there is no "truncated" variant since native inputs don't change their content this way.

<img width="1441" alt="Screenshot 2023-08-23 at 17 17 51" src="https://github.com/metabase/metabase/assets/8542534/a9afb813-6b20-4890-b2c9-e335ba9d58ad">
